src/lib.rs

```diff
@@ -9,7 +9,8 @@
 
 // From rustc.
 extern crate arena;
-#[macro_use] extern crate rustc;
+#[macro_use]
+extern crate rustc;
```

Comment:
IMO, this is actually a good change, for consistency. Attributes usually go above statements. You'd never write:

```rust
#[inline] fn test() {
}
```